# Controller
gain_lambda: 20
gain_nu: 1

# Image Recognition
target: "car" #"sports ball" #"car"
probability: 0.05

typhoon:              
  intrinsic_matrix:
    FL_1x: 1280.784
  relPos_camPlane:
    x: -0.022
    y: 0.000
    z: -0.212
  relPos_panPlane:  # flu
    x: -0.026
    y: 0.000
    z: -0.1
  relPos_camPan:  # pan
    x: 0.016
    y: 0.000
    z: 0.062

miniyy:               # horizontal_fov: 0.5465373885
  intrinsic_matrix:
    FL_1x: 1280.784     # 6.5 mm
  relPos_camPlane:
    x: 1.160
    y: 0.000
    z: -0.050
  relPos_panPlane:  # flu
    x: 1.156
    y: 0.000
    z: 0.062
  relPos_camPan:  # pan
    x: 0.016
    y: 0.000
    z: 0.062

techpod:
  intrinsic_matrix:
    FL_1x: 1280.784
  relPos_camPlane:
    x: 0.360
    y: 0.000
    z: -0.110
  relPos_panPlane:  # flu
    x: 0.356
    y: 0.000
    z: 0.002
  relPos_camPan:  # pan
    x: 0.016
    y: 0.000
    z: 0.062